includes/views/widget/widget-settings.php

```diff
@@ -0,0 +1,63 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+?>
+<p>
+	<label for="<?php echo esc_attr( $this->get_field_id( 'title' ) ); ?>"><?php _e( 'Title:', 'business-directory-plugin' ); ?></label>
+	<input class="widefat" id="<?php echo esc_attr( $this->get_field_id( 'title' ) ); ?>" name="<?php echo esc_attr( $this->get_field_name( 'title' ) ); ?>" type="text" value="<?php echo esc_attr( $this->get_field_value( $instance, 'title' ) ); ?>" />
+</p>
+<p>
+	<label for="<?php echo esc_attr( $this->get_field_id( 'number_of_listings' ) ); ?>"><?php _e( 'Number of listings to display:', 'business-directory-plugin' ); ?></label>
+	<input class="widefat" id="<?php echo esc_attr( $this->get_field_id( 'number_of_listings' ) ); ?>" name="<?php echo esc_attr( $this->get_field_name( 'number_of_listings' ) ); ?>" type="text" value="<?php echo esc_attr( $this->get_field_value( $instance, 'number_of_listings' ) ); ?>" />
+</p>
+<?php $this->_form( $instance ); ?>
+
+<?php if ( in_array( 'images', $this->supports ) ) : ?>
+
+	<?php $style = 'style="' . ( $this->get_field_value( $instance, 'show_images' ) ? '' : 'display: none;' ) . '"'; ?>
```

Comment:
This makes it tricky to escape the "echo $style" lines later. How about this?
`$style = $this->get_field_value( $instance, 'show_images' ) ? '' : 'display: none';`

Then when it's used:
`style="<?php echo esc_attr( $style ); ?>"`

Or better yet, we should use the "hidden" class.

Reply:
Used `hidden` class and simplified the check on this section